synedrion/src/curve/arithmetic.rs

```diff
@@ -164,8 +164,6 @@ impl<'de> Deserialize<'de> for Scalar {
 
 impl DefaultIsZeroes for Scalar {}
 
-impl DebugSecret for Scalar {}
```

Comment:
I assume we don't need a manual implementation of `DebugSecret` anymore like for `SecretKeyPaillier` since a) this already had a derived `Debug` implementation and b) anybody that's using for secret values these can wrap them in a `SecretBox` and use that `Debug` implementation